### ISSUE TITLE
Preview Dialogue window mixes up variables on the same line

### ISSUE THREAD

**Issue by Cratesmith** [YarnSpinnerTool/VSCodeExtension#41]
**What is the current behavior?**

Variables are correct in both the .yarn file and in the preview window's variables list... however the line displayed in the preview has the variables flipped around somehow.
![image](https://github.com/YarnSpinnerTool/VSCodeExtension/assets/4616107/640cdde7-e452-49d3-b098-19862ed94f97)

**Please provide the steps to reproduce, and if possible a minimal demo of the problem**:

Open the following yarn script in vscode. 
Open the dialogue preview 
Run the Hitchhiker_Interact_AskToLeave node
Observe the variables in the previewed line are mixed up when compared to the node in the yarn file.

```
title: Hitchhiker_Init
tags:
position: -526,-697
---
<<declare $NPC_Destination = "Destination">>
<<declare $NPC_Name = "Hitchhiker">>
<<declare $NPC_HasInteracted = false>>
<<declare $NPC_IsOnCaravan= false>>
===

title: Hitchhiker_Interact_AskToLeave
tags:
position: -862,-562
---
{$NPC_Name}: Hey there! We still headed to {$NPC_Destination}?
-> <rainb>You bet!</rainb>
    {$NPC_Name}: Nice!
-> Nope.
    {$NPC_Name}: Ah damn. Cheers for the trip so far!
    <<set $NPC_IsOnCaravan= false>>
<<set $NPC_HasInteracted=true>>
===
```

**What is the expected behavior?**
Expected the line to run with variables not flipped around

**Please tell us about your environment:**

  - Yarn Spinner Version: v2.2.122 (both pre-release and current release through vscode extensions tab)
  - Extension Version:  v2.3.1 (unrelated)
  - Unity Version:  2023.1.3f1

**Other information**

**Comment by ArthurGodoy** [comment]
Same here (using vscode extension).
@Cratesmith did you find any workaround?

**Comment by Cratesmith** [comment]
Not a great one,
The project I'm working on has a very custom variable storage backend so I am building tools to test/debug dialogue in game.

Fixing this issue would be a godsend for our writer however

-------- Original Message --------

**Comment by desplesda** [comment]
Fixed in 61379e61172fe8f88ae579c74440d9a18ecff066.
